packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -40,22 +49,48 @@ export interface Build {
   fragments: Fragment[];
 }
 
+// projectName => { filePath => document }
```

Comment:
🔥

Reply:
yea probably don't need that any longer, it was helpful during the build phase 😁